package/yast2.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Thu Jan  3 11:37:36 UTC 2019 - dgonzalez@suse.com
+
+- Fetch (and display) properly the license content when using a
+  product definition located in /usr/share/installation-products
```

Comment:
I think we should mention here, that it is location inside rpm package and not in insts-sys. At least if I get how code works.

Reply:
You are right. Thank you.